### PR TITLE
Add support for version command

### PR DESCRIPTION
**Pull request by lkysow**
How I've tested this PR:
- compiled and run

How I expect reviewers to test this PR:
- compile and run

Checklist:
- Tests added: No tests added. This matches what we're doing in control-plane (no tests)
- CHANGELOG entry added

### CODE REVIEW

File: cli/cmd/version/version.go [hunk 1]
```diff
@@ -2,49 +2,32 @@ package version
 
 import (
 	"fmt"
-	"strings"
-)
+	"sync"
 
-var (
```
Comment (lkysow) [inline cli/cmd/version/version.go:8]: this code moved to `version/version` in top-level

File: control-plane/subcommand/version/command.go [hunk 2]
```diff
@@ -12,7 +12,7 @@ type Command struct {
 }
 
 func (c *Command) Run(_ []string) int {
-	c.UI.Output(fmt.Sprintf("consul-k8s %s", c.Version))
+	c.UI.Output(fmt.Sprintf("consul-k8s-control-plane %s", c.Version))
```
Comment (lkysow) [inline control-plane/subcommand/version/command.go:15]: fix missing change from monorepo move

**Review by ndhanushkodi — APPROVED** [review]
Thanks for adding this, looks great! 

We did have one changelog entry that went into the alpha release (only install was merged at the time) https://github.com/hashicorp/consul-k8s/blob/main/CHANGELOG.md#0340-september-17-2021

So this could be added to the changelog as support for version command. 

I think the changelog is missing the uninstall command as well

File: cli/cmd/version/version.go [hunk 3]
```diff
@@ -2,49 +2,32 @@ package version
...
+}
+
+func (c *Command) Run(_ []string) int {
+	c.once.Do(c.init)
+	c.UI.Output(fmt.Sprintf("consul-k8s %s", c.Version))
+	return 0
+}
+
+func (c *Command) Synopsis() string {
+	return "Prints the version."
```
Comment (t-eckert) [inline cli/cmd/version/version.go:28]:
```suggestion
	return "Prints the version of the CLI."
```

File: cli/cmd/version/version.go [hunk 4]
```diff
@@ -2,49 +2,32 @@ package version
...
+}
+
+func (c *Command) Help() string {
+	return ""
```
Comment (t-eckert) [inline cli/cmd/version/version.go:32]: Should there be something returned here?

**Review by t-eckert — APPROVED** [review]
Just a few comments

File: cli/cmd/version/version.go (hunk 4 above)
Comment (lkysow) [reply cli/cmd/version/version.go:32]: yeah good call, I copied https://github.com/hashicorp/consul/blob/main/command/version/version.go#L90

**Pull request merged**